database/migrations/2022_03_12_135619_create_accounting_periods_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAccountingPeriodsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('accounting_periods', function (Blueprint $table) {
+            $table->id();
+             $table->date('month');
+            $table->date('year');
```

Comment:
How about we'll changing this to `date_from` and `date_to`, Sir @justinmanigo  ? Since accounting periods will be based on Ethiopian Calendar so this initial schema intended for monthly accounting periods will no longer be necessary.